helios-client/src/main/java/com/spotify/helios/common/descriptors/RolloutOptions.java

```diff
@@ -165,39 +175,31 @@ public String toString() {
 
   public static class Builder {
 
-    private long timeout;
-    private int parallelism;
-    private boolean migrate;
-    private boolean overlap;
+    private Long timeout;
+    private Integer parallelism;
+    private Boolean migrate;
+    private Boolean overlap;
     private String token;
-    private boolean ignoreFailures;
-
-    public Builder() {
```

Comment:
I changed the behavior of this `Builder` to not set defaults for fields since I wanted the `rollingUpdateCommand` to be able to send a partially filled out set of options. An alternative approach might be to preserve the "builder gives defaults" pattern here and change [this line](https://github.com/spotify/helios/pull/1165/files#diff-09ebe80c4b7d83ee0a1323aed194d6baR167) to use the raw constructor.

Reply:
@caipre Your link didn't go to a line. Are you referring to [this line](https://github.com/spotify/helios/pull/1165/files#diff-48145829c6492a8c9e879873420922afR106)?

Reply:
Above this line a bit: https://github.com/spotify/helios/pull/1165/files#diff-09ebe80c4b7d83ee0a1323aed194d6baR172